src/extrinsic/extra.rs

```diff
@@ -230,13 +230,16 @@ impl SignedExtension for ChargeTransactionPayment {
 pub trait SignedExtra<T: Config>: SignedExtension {
     /// The type the extras.
     type Extra: SignedExtension + Send + Sync;
+    /// The extra config parameters.
+    type Config: Default + Send + Sync;
```

Comment:
Nit: I was initially a bit confused as there is also a `Config` trait above. Perhaps this should be called osmething more unique (and vague) like `Other`, since presumably its goal is simply to allow other arbitrary data to be provided?

Reply:
How about "Parameters"?

Reply:
Parameters sounds better. Renamed.